### ISSUE TITLE
Fix CI for MW 1.38 (aria-disabled='false' removed)

### ISSUE THREAD

**Issue by gesinn-it-gea** [SemanticMediaWiki/SemanticDrilldown#45]
Json Test Case Script Runner: Failed "no filter applied"

```
│ ==== (StringContains expected) ====
│ [ <input type='date' tabindex='0' aria-disabled='false' name='_lower_Date' value='2000-01-01' placeholder='YYYY-MM-DD' class='oo-ui-inputWidget-input' /> ], [ <input type='date' tabindex='0' aria-disabled='false' name='_upper_Date' value='2000-01-02' placeholder='YYYY-MM-DD' class='oo-ui-inputWidget-input' /> ]
│
│ Failed asserting that 20 matches expected 22.
```

**Comment by gesinn-it-gea** [comment]
This is probably caused by the missing `aria-disabled='false'` [introduced by OOUI v0.43.0](https://gerrit.wikimedia.org/g/oojs/ui/+/v0.43.0/History.md):

> Widget: Minimize DOM by not adding default aria-disabled="false" (Thiemo Kreuz)
